CONTRIBUTORS

```diff
@@ -0,0 +1,11 @@
+Copyright 2018-2019 The sticker contributors
+
+sticker is provided to you by its contributors under the Blue Oak Model License
+Version 1.0.0 (see LICENSE.md). Both Daniël de Kok and Tobias Pütz may
+individually, at their own discretion, relicense future versions of sticker
+under any later version of the Blue Oak Model License.
+
+Contributors:
+
+Daniël de Kok <me@danieldk.eu>
+Tobias Pütz <tobias.puetz@student.uni-tuebingen.de>
```

Comment:
Perhaps change to tobias.puetz@uni-tuebingen.de which may be the more permanent address.

Reply:
Good idea! Fixed.